stockfish/models.py

```diff
@@ -20,6 +21,9 @@ class StockfishException(Exception):
 class Stockfish:
     """Integrates the Stockfish chess engine with Python."""
 
+    del_counter = 0
```

Comment:
maybe it would be better to call it that?

```suggestion
    _del_counter = 0
```